libuavcan/include/libuavcan/time.hpp

```diff
@@ -144,7 +181,14 @@ class Base
 
     Type operator-() const
     {
-        return fromMicrosecond(-usec_);
+        if (usec_ == std::numeric_limits<USecT>::max())
+        {
+            return fromMicrosecond(std::numeric_limits<USecT>::min() + 1);
+        }
+        else
+        {
+            return fromMicrosecond(-usec_);
+        }
```

Comment:
Given that for a two's complement signed integer `abs(min) == max + 1`, does this code handle the case where `usec_ == std::numeric_limits<USecT>::min()` correctly? I couldn't find this case covered in the unit tests but maybe I just missed it.

Reply:
erp. Fixed this. Thanks.